README.md

```diff
@@ -9,11 +9,11 @@
 
 # TYPO3 Console
 
-TYPO3 Console (helhum/typo3-console) provides a clean way to register commands and
+TYPO3 Console (`helhum/typo3-console`) provides a clean way to register commands and
 a sane way to call these commands through the TYPO3 core command line tool called `typo3`.
 
 ### Important information for longtime users
-**Since version 8.0.0 the extra binary `typo3cms` is removed and superseded with the core binary `typo3`.
+**Since version 8.0.0, the extra binary `typo3cms` is removed and superseded with the core binary `typo3`.
```

Comment:
```suggestion
**The CLI binary `typo3cms` was removed with version 8.0.0 and is replaced by the TYPO3 CLI binary `typo3`.
```

Reply:
@oliverklee any reason you marked this as resolved? I actually find @mbrodala version much more accessible

Reply:
Oh, I'm sorry, I did not intend to do that. I think I misclicked or forgot to take it. Yes, please let's have the change.